CONTRIBUTING.md

```diff
@@ -1,15 +1,20 @@
 Thanks for contributing to `nvm`!
 
-We love pull requests, they're our favorite.
+We love pull requests and issues, they're our favorite.
 
 However, before submitting, please review the following:
 
+For bug reports:
+ - Please make sure the bug is reproducible, and give us the steps to reproduce it, so that we can dig into the problem.
+ - Please give us the info about your environments as detail as possible, so that we can confirm the problem easier.
```

Comment:
"Please give us as much detail as possible about your environment, so we can more easily confirm the problem"